aten/src/ATen/templates/DispatchKeyFunctions_inl.h

```diff
@@ -7,6 +7,14 @@
 #include <c10/core/Scalar.h>
 #include <ATen/core/Reduction.h>
 
+// TODO: If necessary, consider adding <ATen/ops/{function}_key.h> headers
```

Comment:
Is this for you or future devs?

Do we want to write a "note" somewhere about the function specific headers and have all the error message point to it?

Reply:
Sorry, just noticed my comment didn't get posted. This was a note to myself, referring to splitting this header into per-operator headers. The next PR in the stack resolves that and the comment is gone.